settings.gradle

```diff
@@ -2,6 +2,7 @@ rootProject.name = 'arrow-fx-lib'
 
 include 'arrow-fx'
 include 'arrow-fx-coroutines'
+include 'arrow-fx-coroutines-kotlinx'
```

Comment:
arrow-fx-coroutines-kotlinx vs arrow-fx-kotlinx-coroutines 🤔

Reply:
Haha, ye it's a pickle.
I didn't want to go `arrow-fx-coroutines-kotlinx-coroutines`, I felt that this indicated more clearly it's part of the `arrow-fx-coroutines` family.

Reply:
I think we should reconsider the naming, otherwise I believe it might cause confusion among users

Reply:
Or could it make sense to add this feature in the existing `arrow-fx-coroutines`?

Reply:
@aballano I'm not sure how this would cause confusion? Due to omitting the final `-coroutines` from `kotlinx`?
We cannot use `arrow-fx-kotlinx-coroutines` since it's already taken by the existing [`IO` integration](https://github.com/arrow-kt/arrow-fx/tree/master/arrow-fx-kotlinx-coroutines).

So I think our only other option would be `arrow-fx-coroutines-kotlinx-coroutines` which I thought was a bit redundant. Better suggestions are welcome.

@rachelcarmena We don't want to include KotlinX Coroutines into the `arrow-fx-coroutines` module since both offer different concurrent implementations for the std's coroutine system.

Reply:
Maybe something including the wording "integration-kotlinx" somewhere so we're clear that this is an extra **for** that specific lib 👌

Reply:
None of the other libraries of Arrow Fx use such naming at the moment, wouldn't it cause more confusion to break the current naming patterns we've been using?

I've also not seen this commonly done for any other effects libraries. I.e. the integrations for KotlinX also just post-fix
https://github.com/Kotlin/kotlinx.coroutines/tree/master/reactive
https://github.com/Kotlin/kotlinx.coroutines/tree/master/integration

Reply:
Others integrations need to follow soon such as:

- `arrow-fx-coroutines-rxjava`
- `arrow-fx-coroutines-reactor`

They're needed to cover the functionality we're deprecated from the Arrow Fx IO library.

Reply:
I prefer `arrow-fx-coroutines`-xxx since its part of the family 👍